flask_main.py

```diff
@@ -17,6 +18,14 @@ def resturant_route():
 
     return RestaurantService().get_restaurant_in_area(lat=lat, long=long,num_results=num_results)
 
+# compares 2 restaurants
+@app.route("/restaurants/compare", methods=['GET', 'POST'])
```

Comment:
change methods=['GET','POST'] to methods=['GET'],

this code snippet defines what type of request calls can be made to this route, POST calls are for creating data but we're just returning the similarity score / comparing two resturants by ids